src/main/java/com/laborete/LaboreteAPI/posts/service/PostsService.java

```diff
@@ -11,5 +11,8 @@ public interface PostsService {
     PostDTO createPost(CreatePostDTO post);
 
     PostDTO getPostById(UUID id);
+
+    List<PostDTO> getFilteredPosts(String text);
```

Comment:
I would simplify the name of the method - `filterPosts`. The current name is good. There is a convension to name a method starting with a verb. Your name follows this rule - _get_. But `filterPosts` sounds a bit more clear to me - we emphasize that the method does filtering of posts inside.

And also, the method should take an instance of type `FilterDto` as a param now.